react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+export default function StopWatch() {
+  const switchClasses = {
+    play: {
+      icon: 'fa-play',
+      playColor: 'green'
+    },
+    pause: {
+      icon: 'fa-pause',
+      playColor: 'red'
+    },
+    count: 0
+  };
+
+  let [start, setStart] = useState(null);
+  const [toggle, setToggle] = useState(false);
+  let [time, setTime] = useState(0);
+  function handlePlayClick() {
+    clearInterval(start);
```

Comment:
It's safest to only call `clearInterval` if `start` is a valid ID returned from `setInterval`. MDN doesn't say what happens if you call `clearInterval` with an invalid ID, so it probably works, but it's safest to be careful.

This also means that when you clear the interval you need to unset `start` so that it always has a valid value. Note that if you do that, you can use `start` to determine if you are playing and can get rid of the `toggle` state (since you'd be able to calculate it from `start`.